model/file_info_test.go

```diff
@@ -22,179 +24,127 @@ func TestFileInfoIsValid(t *testing.T) {
 		Path:      "fake/path.png",
 	}
 
-	if err := info.IsValid(); err != nil {
-		t.Fatal(err)
-	}
+	err := info.IsValid()
+	require.Nil(t, err)
 
 	info.Id = ""
-	if err := info.IsValid(); err == nil {
-		t.Fatal("empty Id isn't valid")
-	}
+	err = info.IsValid()
+	require.NotNil(t, err, "empty Id isn't valid")
 
 	info.Id = NewId()
 	info.CreateAt = 0
-	if err := info.IsValid(); err == nil {
-		t.Fatal("empty CreateAt isn't valid")
-	}
+	err = info.IsValid()
+	require.NotNil(t, err, "empty CreateAt isn't valid")
 
 	info.CreateAt = 1234
 	info.UpdateAt = 0
-	if err := info.IsValid(); err == nil {
-		t.Fatal("empty UpdateAt isn't valid")
-	}
+	err = info.IsValid()
+	require.NotNil(t, err, "empty UpdateAt isn't valid")
 
 	info.UpdateAt = 1234
 	info.PostId = NewId()
-	if err := info.IsValid(); err != nil {
-		t.Fatal(err)
-	}
+	err = info.IsValid()
+	require.Nil(t, err)
 
 	info.Path = ""
-	if err := info.IsValid(); err == nil {
-		t.Fatal("empty Path isn't valid")
-	}
+	err = info.IsValid()
+	require.NotNil(t, err, "empty Path isn't valid")
 
 	info.Path = "fake/path.png"
-	if err := info.IsValid(); err != nil {
-		t.Fatal(err)
-	}
+	err = info.IsValid()
+	require.Nil(t, err)
 }
 
 func TestFileInfoIsImage(t *testing.T) {
 	info := &FileInfo{
 		MimeType: "image/png",
 	}
 
-	if !info.IsImage() {
-		t.Fatal("file is an image")
-	}
+	assert.False(t, !info.IsImage(), "file is an image")
```

Comment:
double negation? why not use assert.True?

Reply:
Fair enough